src/components/controls/MediaStreamControl.tsx

```diff
@@ -12,10 +24,23 @@ export class MediaStreamControl extends React.PureComponent<IMediaControlProps,
 
         this.onMuteButtonClick = this.onMuteButtonClick.bind(this);
         this.onDisableButtonClick = this.onDisableButtonClick.bind(this);
+        this.getAudioEnabled = this.getAudioEnabled.bind(this);
+        this.setAudioEnabled = this.setAudioEnabled.bind(this);
+        this.getVideoEnabled = this.getVideoEnabled.bind(this);
+        this.setVideoEnabled = this.setVideoEnabled.bind(this);
     }
 
     // TODO(yunsi): Accept JSX.Element from props and let user customize the button style or button label.
```

Comment:
TODO is done by this change, right? If so, remove the TODO

Reply:
Done